### PR TITLE
#34 create layout

### PR DESCRIPTION
**Pull request by ipieropan**

### CODE REVIEW

**Review by ipieropan — COMMENTED** [review]
Now it's done, next step is the styling. 

`Header` and `Footer` will be called on `Index.js` and `<app />` will be between them, instead of creating a Layout and wrap it.

**Review by maqsoodulhasan — APPROVED** [review]
working

**Review by gamo-code — CHANGES_REQUESTED** [review]
Refactor and change according to https://github.com/academind/react-complete-guide-code/tree/zz-reactjs-summary/code/19-finished

File: Gladiator.Presentation.Ui/src/Components/Layout/Layout.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Header from "./Header";
+import Footer from "./Footer";
+
+
+function Layout(props) {
+    return (
+      <div>
```
Comment (gamo-code) [inline Gladiator.Presentation.Ui/src/Components/Layout/Layout.js:7]: apply bootstrap container class to div.

**Review by gamo-code — CHANGES_REQUESTED** [review]
Create tests for layout footer header

**Review by gamo-code — APPROVED** [review]

**Pull request merged**